tests/src/test/scala/com/nvidia/spark/rapids/shuffle/RapidsShuffleIteratorSuite.scala

```diff
@@ -148,7 +148,7 @@ class RapidsShuffleIteratorSuite extends RapidsShuffleTestHelper {
       123))
 
     val ac = ArgumentCaptor.forClass(classOf[RapidsShuffleFetchHandler])
```

Comment:
The `handler` extracted from this is unused, wondering if there's any reason to capture at all or if there's a missing check on `handler`.